projects/igniteui-angular/chat-extras/src/markdown-service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@angular/core';
+import { Marked } from 'marked';
+import markedShiki from 'marked-shiki';
+import { bundledThemes, createHighlighter } from 'shiki/bundle/web';
```

Comment:
I believe these should (besides being documented) be at least listed as some form of dependency (optional peer?) for the library package if the source imports from them.